lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app-helm/helm/lighty-rnc-app-helm/values.yaml

```diff
@@ -23,10 +23,15 @@ lighty:
     restconfPort: 8888
     restconfPath: "/restconf"
     useHttps: false
+    needClientAuth: false
     keyStorePassword: "8pgETwat"
     keyStoreType: "JKS"
     keyStoreDirectory: "keystore"
     keyStoreFileName: "lightyio.jks"
+    trustKeyStorePassword: "8pgETwat"
```

Comment:
I have concerns: https://kubernetes.io/docs/concepts/configuration/secret/

Reply:
What is your concern about it? One needs to hide values. yaml. This is an environment file as far as I see and it exists to hide these kind of secrets in it.

Reply:
Any progress for the PR approval?

Reply:
IMO: using https://kubernetes.io/docs/concepts/configuration/secret/ we can limit the users which can view password, for example when viewing running configuration.

I would be glad if you try to create secret as described in the link above. In case you do not have time for this, please tell us and @PeterSuna can take care of it.

Thank you for you contribution!

Reply:
@ihrasko @PeterSuna Thanks for your comments. First of all I do not have much time this week and secondly I do not have sufficient practical experience with kubernetes and helm. So I'd recommend it to be applied at your side, IMHO as another feature or change request. Because it's not only relevant for _trustKeyStorePassword_ but also for _keyStorePassword_.

Reply:
OK, ACK.